### PR TITLE
CIV-6597: DQs - ft1

### PR DESCRIPTION
**Pull request by portizdev**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-6597

### Change description ###
Screens:
- Defendant Response - DQ - FT - Have to tried to settle this claim
- Defendant Response - DQ - FT - Do you want an extra 4 weeks to settle the claim
- Defendant Response - DQ - FT - Are there any documents the claimant has that you want the court to consider?

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by KyleMulvihillV1 — APPROVED** [review]

File: src/test/unit/services/features/directionsQuestionnaire/directionQuestionnaireService.test.ts [hunk 1]
```diff
@@ -230,6 +230,31 @@ describe('Direction questionnaire Service', () => {
       expect(expertEvidence.option).toBe(YesNo.YES);
     });
 
+    it('should return request defendantExpertEvidence option with Yes option - defendant journey', async () => {
+      const claim = new Claim();
```
Comment (TatsyMaslova) [inline src/test/unit/services/features/directionsQuestionnaire/directionQuestionnaireService.test.ts:234]: sorry to be a best but could have a given when then pattern in our tests? to keep it consistent and easier to read

File: src/main/services/features/directionsQuestionnaire/directionQuestionnaireService.ts [hunk 2]
```diff
@@ -23,7 +23,12 @@ const getDirectionQuestionnaire = async (claimId: string): Promise<DirectionQues
 const getGenericOption = async (claimId: string, directionQuestionnairePropertyName: string, parentPropertyName?: string): Promise<GenericYesNo> => {
   try {
     const caseData = await getCaseDataFromStore(claimId);
-    const directionQuestionnaire: any = caseData?.directionQuestionnaire ? caseData.directionQuestionnaire : new DirectionQuestionnaire();
+    let directionQuestionnaire: any;
+    if(caseData.isClaimantIntentionPending()){
+      directionQuestionnaire = caseData.claimantResponse.directionQuestionnaire ? caseData.claimantResponse.directionQuestionnaire : new DirectionQuestionnaire();
```
Comment (laperla11) [inline src/main/services/features/directionsQuestionnaire/directionQuestionnaireService.ts:28]: I would change it line 28 as below to be on safe side;
`directionQuestionnaire = caseData.claimantResponse.?directionQuestionnaire ? caseData.claimantResponse.directionQuestionnaire : new DirectionQuestionnaire();`

**Review by laperla11 — APPROVED** [review]

File: src/test/unit/services/features/directionsQuestionnaire/directionQuestionnaireService.test.ts (hunk 1 above)
Comment (portizdev) [reply src/test/unit/services/features/directionsQuestionnaire/directionQuestionnaireService.test.ts:234]: Sure

File: src/main/services/features/directionsQuestionnaire/directionQuestionnaireService.ts (hunk 2 above)
Comment (portizdev) [reply src/main/services/features/directionsQuestionnaire/directionQuestionnaireService.ts:28]: Done

**Review by RosDiaf — APPROVED** [review]

**Pull request merged**